framer/Animation.coffee

```diff
@@ -190,7 +190,11 @@ class exports.Animation extends EventEmitter
 	_updateValue: (value) =>
 
 		for k, v of @_stateB
-			@_target[k] = Utils.mapRange(value, 0, 1, @_stateA[k], @_stateB[k])
+
```

Comment:
Not in favour of dropping 'new' because the type inferencer will only see a function call, whereas when it's new prefixed, it'll know it is dealing with a constructor.